### PR TITLE
Added UI for status indicators + change to use Node-native https call

### PR DESCRIPTION
**Pull request by DivergentEuropeans**
image

other styles include:

image

Why change to Node's https? Because fetch is not available on Windows in Electron apps

### CODE REVIEW

File: src/renderer/components/Header.tsx [hunk 1]
```diff
@@ -8,22 +8,89 @@
...
+  severity: AlertColor;
+}
+
+// Global event emmitter Zen can use
+export const alertEmitter = new EventEmitter();
 
 export default function Header() {
+  const [alertData, setAlertData] = useState<AlertData>({
+    show: false,
+    duration: 5000,  // Default 5 seconds
```
Comment (DivergentEuropeans) [inline src/renderer/components/Header.tsx]: Change default to not go away

File: src/actions/PlanningActions.ts [hunk 2]
```diff
@@ -41,30 +42,80 @@ export class PlanningActions {
...
+  // public static async getExampleZowe() {
+  //   try {
+  //     const exampleZowe = await fetch('https://raw.githubusercontent.com/zowe/zowe-install-packaging/v2.x/master/example-zowe.yaml');
+  //     const exampleZoweText = await exampleZowe.text();
+  //     ConfigurationStore.setConfig(parse(exampleZoweText));
+  //     return {status: true, details: parse(exampleZoweText)};
+  //   } catch (error) {
+  //     return {status: false, details: {error}};
+  //   }
+  // }
```
Comment (DivergentEuropeans) [inline src/actions/PlanningActions.ts]: remove comment

File: src/actions/PlanningActions.ts [hunk 3]
```diff
@@ -41,30 +42,80 @@ export class PlanningActions {
...
+            reject({status: false, details: {error}});
+          }
+        });
+      }).on('error', (error) => {
+        reject({status: false, details: {error}});
+      });
+    });
   }
 
+  // public static async getZoweSchema() {
```
Comment (DivergentEuropeans) [inline src/actions/PlanningActions.ts]: remove comment

File: src/renderer/components/Header.tsx [hunk 4]
```diff
@@ -8,22 +8,89 @@
...
     <React.Fragment> 
       <Box>
         <AppBar position="static">
           <Toolbar sx={{boxShadow: '0px 0px 3px 3px rgb(0 0 0 / 25%)'}}>
-            <Typography variant="h6" component="div" sx={{ flexGrow: 1 }}>
+            <Typography style={{ textAlign: 'right' }} variant="h6" component="div" sx={{ flexGrow: 1 }}>
               Zowe Enterprise Necessity
             </Typography>
+            {alertData.show && (
+              <><Alert style={{ position: 'absolute' }} severity={alertData.severity}>{alertData.message}
```
Comment (DivergentEuropeans) [inline src/renderer/components/Header.tsx:90]: Needs to be an array

UX - how to handle stacked statuses that are too long, which one to prioritize, truncate all equally?

Comment (DivergentEuropeans) [reply src/renderer/components/Header.tsx:90]: this comment will be added in a future PR

**Review by DivergentEuropeans — COMMENTED** [review]
So far, this PR covers steps 
1-2

**Review by timgerstel — APPROVED** [review]
Reviewed code on webex, saw demo. Approved.

**Comment by timgerstel** [comment]
Tested, works as expected

**Pull request merged**